### ISSUE TITLE
Improve the guest avatar styling in call view

### ISSUE THREAD

**Issue by DorraJaouad** [nextcloud/spreed#10629]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

**Is your feature request related to a problem? Please describe.**
Guests are identified with a gray avatar  in call view :
![image](https://github.com/nextcloud/spreed/assets/84044328/61e66f6f-0bcc-4d70-97b0-6a57bbd428c0)

This identification method might be okay in other places like messages and participant lists because it doesn't own much of "visual weight" but for call view, it occupies more visual surface. Grayscale stands out as looking visually "disabled" compared to all the other avatars. 

**Describe the solution you'd like**
The guest avatar style is the same as the other avatars ( with the letter to always be uppercase in call view). To keep it distinguishable,  add "( guest )" in the display name.
![image](https://github.com/nextcloud/spreed/assets/84044328/cfb9429d-bc27-4b7f-84d0-a0a5417e91d6)

**Comment by nickvergessen** [comment]
Isn't it confusing if the user has 2 different avatars in 2 places?

**Comment by DorraJaouad** [comment]
Yes, might be but still is a bit incohesive to keep it like that in the call view.
This implies another suggestion :
- change the letter color to the background color so it can follow at least the same pattern.
AND
-  make it uppercase ( you can see that user accounts have their names in lower cases but their avatars are capitalized)

**Comment by jancborchardt** [comment]
Then for both keeping the separation, but also making it look good:
- Keep the grey circle as @nickvergessen mentions for separation
- Instead of the initial, use this icon: https://pictogrammers.com/library/mdi/icon/account/
- Append "(Guest)" to the name as @DorraJaouad mentioned (no spaces around the word "Guest")

What do you think?

**Comment by Antreesy** [comment]
We're also using this component (and same styles) for representing the guest as messages author or in participant list. Should we handle these cases the same? (person icon + Guest badge)

**Comment by nickvergessen** [comment]
They should have the `(Guest)` in all the cases already

**Comment by DorraJaouad** [comment]
Suggestion 3 : 

![image](https://github.com/nextcloud/spreed/assets/84044328/3548256e-15b7-486a-a673-ac7ead13f822)

**Comment by nickvergessen** [comment]
Should use the same circle color as normal users? 
The grey circle is from the old times when the circle was colored and the letter white/black.

So to align a bit more the background could now also be that white/black backrgound color and the icon in the user color like on your screenshot?

**Comment by Antreesy** [comment]
IMO better than the question mark
